packages/elements/src/slider/elements/slider-marker.ts

```diff
@@ -0,0 +1,58 @@
+import { BasicElement, CSSResultGroup, TemplateResult, css, html } from '@refinitiv-ui/core';
+import { customElement } from '@refinitiv-ui/core/decorators/custom-element.js';
+import { property } from '@refinitiv-ui/core/decorators/property.js';
+
+import { VERSION } from '../../version.js';
+
+@customElement('ef-slider-marker')
+export class SliderMarker extends BasicElement {
+  /**
+   * Element version number
+   * @returns version number
+   */
+  static override get version(): string {
+    return VERSION;
+  }
+
+  static override get styles(): CSSResultGroup {
+    return css`
+      :host {
+        position: absolute;
+        z-index: 2;
```

Comment:
It seems if we move `<div part="track-fill"...>` under `<div part="step-container"...>` in `renderTrack()`, there would be no need to set `z-index` at all. I did confirm visually + I can't see any other impacts.